keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -126,13 +141,3 @@ def get_initial_parameters(self, timesteps, batch_size, seed=None):
             (batch_size, self.img_height // 8, self.img_width // 8, 4), seed=seed
         )
         return noise, alphas, alphas_prev
-
```

Comment:
Feel free to add this to examples/models/stable_diffusion/

right now they’re basically glorified debugging demos; but we could render them someday down the line with proper polish